lib/android/app/src/main/java/com/reactnativenavigation/options/BackButton.java

```diff
@@ -46,7 +46,7 @@ public boolean hasValue() {
 
     public void mergeWith(BackButton other) {
         if (!Constants.BACK_BUTTON_ID.equals(other.id)) id = other.id;
-        if (other.accessibilityLabel.hasValue()) accessibilityLabel = other.accessibilityLabel;
+        if (other.accessibilityLabel.hasValue() && !other.accessibilityLabel.equals(new Text("Navigate Up"))) accessibilityLabel = other.accessibilityLabel;
```

Comment:
Seem when user push a new screen `addToPushedChild` invoke `mergeOptions` with default options 


![image](https://user-images.githubusercontent.com/4661784/135737804-28ea6057-2787-445a-ad86-6f4d350aeaf9.png)

so, the custom value will be overwritten with a default value 
```java
this === child.options.topBar.buttons.back // My custom value ...
other === options.topBar.buttons.back  // Navigate Up

if (other.accessibilityLabel.hasValue()) accessibilityLabel = other.accessibilityLabel;
```